lib/tasks/queue/rspec.rake

```diff
@@ -5,7 +5,7 @@ require 'knapsack_pro'
 namespace :knapsack_pro do
   namespace :queue do
     task :rspec, [:rspec_args] do |_, args|
-      Kernel.exec("RAILS_ENV=test RACK_ENV=test #{$PROGRAM_NAME} 'knapsack_pro:queue:rspec_go[#{args[:rspec_args]}]'")
+      Kernel.exec({ 'RAILS_ENV' => 'test', 'RACK_ENV' => 'test' }, $PROGRAM_NAME, "knapsack_pro:queue:rspec_go[#{args[:rspec_args]}]")
```

Comment:
@ArturT @3v0k4 we dropped the single quotes in this, which could cause some weirdness

Reply:
If you are referring to the missing `'` in `'knapsack_pro:queue:rspec_go[#{args[:rspec_args]}]'`, that was left out intentionally.

Previously, the quotes were used to ensure the arguments were parsed correctly. However, since we now pass `"knapsack_pro:queue:rspec_go[#{args[:rspec_args]}]"` as a positional argument, they are no longer necessary. Argument parsing still works, for example, running: `bundle exec rake "knapsack_pro:queue:rspec[--format d --no-color]"` will use the documentation format and disable color in the output.

Reply:
Hmm, yea it should be getting invoked right. I'd love to have repro steps for #301, it's frustrating attempting to figure out what's happening from very little info :(